src/components/Controls.svelte

```diff
@@ -22,17 +22,17 @@
 		}
 	}, 500);
 
-	enum status{
-		PLAY,
-		PAUSE
-	};
-	let currStatus = status.PAUSE;
+	const status = {
+		PLAY: 'play',
+		PAUSE: 'pause'
+	} as const;
+	let currStatus = 'pause';
```

Comment:
nit: what you previously had was fine, and start constants with a capitalized letter. This is non-blocking and can be updated with the next CR.

`let currStatus = Status.PAUSE`

Reply:
resolve conversation and merge!